App/Components/Questionnaire.js

```diff
@@ -1,114 +1,91 @@
 import React, {
-  AppRegistry,
   Component,
-  StyleSheet,
   Text,
-  TextInput,
   View,
   TouchableHighlight,
   Navigator
 } from 'react-native';
 
-import Header from './Header';
 import Question from './Question';
 
-const questionCollection = [{
+const questionCollection = [
+  {
     header: 'Recent HIV & STD Tests',
     questions: [
```

Comment:
just something to think about, since these questions and their answers are going to be returned and handled by the server, and potentially used in multiple components, maybe we should abstract them outside of this component? Your call Hou, I'm not too sure.